target/linux/ramips/image/mt76x8.mk

```diff
@@ -666,6 +666,16 @@ define Device/widora_neo-32m
 endef
 TARGET_DEVICES += widora_neo-32m
 
+define Device/hak5_wifi-pineapple-mk7
+  BLOCKSIZE := 64k
+  IMAGE_SIZE := 32448k
+  DEVICE_VENDOR := Hak5
+  DEVICE_MODEL := WiFi Pineapple Mark 7
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ohci
+  SUPPORTED_DEVICES := wifi-pineapple-mk7
```

Comment:
Is this for factory? If yes, please use +=, if not, drop it.

Reply:
This is to maintain compatibility with devices already running the factory firmware, yes. I'll change to +=.